### PR TITLE
[Intermediate React-Js]: I want to add Dynamic React Dashboard 

### PR DESCRIPTION
**Pull request by inkerton**
Closes: #897 

- **Title:** [Intermediate React-Js]: I want to add Dynamic React Dashboard 
- **Name:** inkerton
- **Idenitfy yourself:** GSSOC

### Describe the add-ons or changes you've made 📃
I added the dynamic dashboard project.

### Screenshots 📷

image
image
[Link](https://www.loom.com/share/23d7e7dbc68640119c32007e737c052b?sid=16f3f673-d4e7-471d-a20b-e0e4d1cd037f)

### Type of change ☑️

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### Checklist: ☑️

- [x] My code follows the [Contributing Guidelines](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CONTRIBUTING.md) & [Code of Conduct](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CODE_OF_CONDUCT.md) of this project.
- [x] This PR does not contain plagiarized content.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x] My changes generate no new warnings.

### CODE REVIEW

**Comment by inkerton** [comment]
@Avdhesh-Varshney What changes do you required can you specify?

File: React-JS-Projects/Intermediate/Dynamic Dashboard/README.md
Comment (Avdhesh-Varshney) [inline React-JS-Projects/Intermediate/Dynamic Dashboard/README.md:1]: Follow readme template

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]
Add 1 ss file and enlist your project in Reactjs readme file by reading the guidelines properly

**Comment by inkerton** [comment]
@Avdhesh-Varshney  I have made the requested changes

**Comment by inkerton** [comment]
@Avdhesh-Varshney Are there more changes Required?

**Comment by Avdhesh-Varshney** [comment]
It can't be approved. Don't create more than 10 commits and not follow guidelines.
- Never commit directly in `main` branch.

**Pull request closed without merging**